### ISSUE TITLE
Add Deploy Source Set for Smart Contract Projects

### ISSUE THREAD

**Issue by mialbu** [neow3j/neow3j#718]
A typical Java contract project consists of a `main` and `test` source set. Contract code goes into `main` and test code into `test`. These source sets are standard for any Java project. For a smart contract project, a third source set - `deploy` - would be convenient. That source set contains all code for deployment on private, test and main net. The IntelliJ IDE can pick up configured source sets from the Gradle build tool and integrate them in the UI and IDE workflows. We could add the `deploy` source set via the already existing neow3j Gradle plugin.

**Comment by csmuller** [comment]
Yes, i like this feature. But as far as the planning goes, this will not fit into June anymore.

**Comment by mialbu** [comment]
> Yes, i like this feature. But as far as the planning goes, this will not fit into June anymore.

I agree. 👍

**Comment by csmuller** [comment]
Thinking about it again: The source set should no be called `deploy` because it will will probably contain other code that is not deployment related, e.g., invocation scripts that handle contract invocations. What would be a good name for such a source set?
